website/docs/installation/windows/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 3
+---
+
+# Windows
+Running Tabby on Windows without WSL or Docker.
+
+## 1. Find the Windows release
+* Go to the Tabby release page: https://github.com/TabbyML/tabby/releases
+* Click on the **Assets** dropdown for a specific release to find the Windows executable (exe) files.
+![Windows release](./win-release.png)
+
+## 2. Download the release
+* If you are using a CPU-only system, download the **tabby_x86_64-windows-msvc.exe file**.
+* If you are using a GPU-enabled system, download the **tabby_x86_64-windows-msvc-cuda{VERSION}.exe** file, replacing {VERSION} with the appropriate CUDA version for your system.
+
+**Tips:**
+* Download the CUDA Toolkit from Nvidia: https://developer.nvidia.com/cuda-toolkit
+* Check your local CUDA version by running the following command in a command prompt or PowerShell window:
+```
+nvcc --version
+```
+
+## 3. Running Tabby locally
+Open a command prompt or PowerShell window in the directory where you downloaded the Tabby executable. Run the following command:
+```
+.\tabby_x86_64-windows-msvc-cuda{VERSION}.exe serve --device cuda
+```
```

Comment:
add more steps to verify installation success (e.g open swagger page and run some inference)